src/components/userComponents/emailPopup/page.tsx

```diff
@@ -0,0 +1,198 @@
+'use client';
+
+import React, { Fragment, useState } from 'react';
+import { Dialog, Transition } from '@headlessui/react';
+import { PiPaperPlaneTiltBold , PiSealCheck } from "react-icons/pi";
+import { VscClose } from "react-icons/vsc";
+import { BiErrorCircle } from "react-icons/bi";
+
+import Link from 'next/link';
+
+
+/**
+ * @returns an email pop up
+ */
+export default function EmailPopup(
```

Comment:
feat:  After NPO feedback, we are going to be using EmailPopup in more places.  Lets add a parameter called "backLink" or something.  We can use this to define where a popup redirects.  Ask/text me for details if confused, I'll add another comment below of what I'm talking about.